src/logic/Preferences.swift

```diff
@@ -197,6 +197,7 @@ enum ShowOnScreenPreference: String, CaseIterable, MacroPreference {
         switch self {
             case .active: return NSLocalizedString("Active screen", comment: "")
             case .includingMouse: return NSLocalizedString("Screen including mouse", comment: "")
+            case .first return NSLocalizedString("Screen including menu bar", comment: "")
```

Comment:
You forgot to add the following line, on line 194:

```swift
case includingMenubar = "Screen including menu bar"
```

Reply:
thanks!